jetstream/lifestyles-images-onboarding-experiment-v3.toml

```diff
@@ -0,0 +1,54 @@
+## EXPERIMENT SPECIFIC
+[experiment]
+segments = ['attributed', 'non_attributed']
+
+start_date = "2023-06-14"
+
+end_date = "2023-06-27"
```

Comment:
What are the actual dates for start, end of enrolment, and end of observation period or the experiment? Two weeks total seems extremely short.

Reply:
I have fixed it